trigger/pom.xml

```diff
@@ -20,87 +20,107 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <parent>
-        <artifactId>parent-conf</artifactId>
-        <groupId>za.co.absa.hyperdrive</groupId>
-        <version>1.0.0-SNAPSHOT</version>
-        <relativePath>../parent-conf/pom.xml</relativePath>
-    </parent>
-    <modelVersion>4.0.0</modelVersion>
 
+    <modelVersion>4.0.0</modelVersion>
+    <version>0.0.1-SNAPSHOT</version>
+    <groupId>za.co.absa.hyperdrive</groupId>
     <artifactId>trigger</artifactId>
 
+    <properties>
+        <scala.compat.version>2.11</scala.compat.version>
+        <spark.version>2.4.1</spark.version>
+    </properties>
+
+
     <dependencies>
         <dependency>
-            <groupId>za.co.absa</groupId>
-            <artifactId>abris_2.11</artifactId>
-            <version>${abris.version}</version>
+            <groupId>com.typesafe.slick</groupId>
+            <artifactId>slick_2.11</artifactId>
```

Comment:
Defining Scala version as a property might be helpful to version management. Something like `scala.compat.version` and then here `slick_${scala.compat.version}`

Reply:
Yes, fixed